jerry-ext/include/jerryscript-ext/arg.impl.h

```diff
@@ -242,6 +243,51 @@ jerryx_arg_string (char *dest, /**< pointer to the native char array where the r
   };
 } /* jerryx_arg_string */
 
+/**
+ * Create a validation/transformation step (`jerryx_arg_t`) that expects to
+ * consume one `string` JS argument and stores it into a C utf8 `char` array.
+ *
+ * @return a jerryx_arg_t instance.
+ */
+static inline jerryx_arg_t
+jerryx_arg_utf8_string (char *dest, /**< pointer to the native char array where the result should be stored */
```

Comment:
`/**< [out] pointer to the native char array where the result should be stored */`